LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ModuleConnectionHost.cs

```diff
@@ -149,6 +159,47 @@ private async Task<MethodResponse> ClearCacheAsync()
             return new MethodResponse((int)HttpStatusCode.OK);
         }
 
+        private async Task<MethodResponse> DropConnectionAsync(MethodRequest methodRequest)
+        {
+            this.forceClosedConnections.Add(1);
+            ReceivedLoRaCloudToDeviceMessage c2d = null;
+
+            this.logger.LogDebug("Will deserialize '{JsonData}'.", methodRequest.DataAsJson);
```

Comment:
too verbose

Reply:
Removed in d489a749d8a517f62eb764a7522091ec36527b1c